app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module Users
+  class RegistrationsController < Devise::RegistrationsController
+    before_action :configure_permitted_parameters
+    before_action :check_token_presence, only: %i[new create]
+
+    after_action :destroy_invitation, only: :create
+
+    def new
```

Comment:
Why do we need this methods? As I see, you just call it without overrides.

Reply:
Actually a think that it is a good Idea to remove an explicit call of new and create, but rubocop throws an error, so I just disable this warning it .rubocop.yml for controllers